starter-code/css/main.css

```diff
@@ -5,6 +5,76 @@ light blue: #1c5380
 */
 
 body {
-  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
-  margin: 0;
+  font: 200 16px/1.5 "open-sans", Helvetica, Arial, sans-serif;
+  padding: 100px 280px; 
+  background: #1e5799;
+  background: #1e5799;
+  background: -moz-linear-gradient(top, #1e5799 43%, #ffffff 43%, #ffffff 46%);
```

Comment:
It was not clear in the guideline but you had an image "bgtile.png", you were supposed to use it as a background image and repeat it on x axe.

If it's not clear ask me (Arthur)